maps/away/ships/dominia/dominian_science_vessel/dominian_science_vessel_ghostroles.dm

```diff
@@ -0,0 +1,133 @@
+/datum/ghostspawner/human/house_volvalaad_voidsman
+	short_name = "house_volvalaad_voidsman"
+	name = "House Volvalaad Voidsman"
+	desc = "You are an enlisted Ma’zal voidsman of the Imperial Fleet, seconded to House Volvalaad for a scientific mission. Voidsmen are generally recruited from the Imperial Frontier and are eager to pay off their Mo’ri’zal (“Blood Debt,” the Empire’s form of taxation) via military service. Few serve more than a few tours of duty due to the lonely and hazardous nature of interstellar military service. You have been trained to obey your superior, and most Primaries, without question or delay. (OOC Note: Players should be familiar with Dominian lore and play a character with a background appropriate to a Dominian Ma'zal.)"
```

Comment:
All of the roles here should have separate `desc` and `welcome_text`.

Description is used in the ghost role spawner window where you select the ghost role to spawn as, and should be a short description of the role.

Welcome text is what you see when you actually spawn, and should include more details, like who you answer to, what is your actual job, what can you do, etc.

![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/c34643a1-7a83-49cd-ac83-63935bbbda00)
see the SCC Scout Ship for an example of t his